traits/ctraits.c

```diff
@@ -4569,17 +4532,9 @@ _trait_property(trait_object *trait, PyObject *args)
 
     if (PyTuple_GET_SIZE(args) == 0) {
         if (trait->flags & TRAIT_PROPERTY) {
-            result = PyTuple_New(3);
-            if (result != NULL) {
-                PyTuple_SET_ITEM(result, 0, temp = trait->delegate_name);
-                Py_INCREF(temp);
-                PyTuple_SET_ITEM(result, 1, temp = trait->delegate_prefix);
-                Py_INCREF(temp);
-                PyTuple_SET_ITEM(result, 2, temp = trait->py_validate);
-                Py_INCREF(temp);
-                return result;
-            }
-            return NULL;
```

Comment:
If we get to this line then `result` is `NULL`, so the `return NULL` is equivalent to `return result`.  So we can (and do) combine the two returns into one in the new code.